packages/gluestick/src/commands/test/__tests__/test.test.js

```diff
@@ -179,5 +188,46 @@ describe('commands/test/test', () => {
       expect(Object.keys(jestConfig.globals).length).toBe(1);
       expect(jestConfig.globals.__DEV__).toBe(true);
     });
+
+    it('should run jest directly with merged default and custom config (no custom moduleNameMapper)', () => {
```

Comment:
I think this is to much copy/paste from the other test. I would write a test case just relevant to the case, so we can define config without custom `moduleNameMapper` and just write a test case that checks that aliases are there nevertheless.